app/extensions/FlyToGeoJsonExtension.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Layer,
+  LayerExtension,
+  UpdateParameters,
+  FlyToInterpolator,
+  WebMercatorViewport,
+} from '@deck.gl/core';
+import { bbox } from '@turf/bbox';
+import { MultiPolygon, MultiPoint } from '~/gen';
```

Comment:
I think it would be better to use Geometries from [`@types/geojson`](https://www.npmjs.com/package/@types/geojson) rather than the `~/gen`

1) The definitions in the `gen` are cobbled together custom implementations. Meanwhile, `@types/geojson` are well-established
2) We should keep the `FlyToExtension` agnostic to the application it is used within. `@types/geojson` supports this independence (while the gen folder is tied to applications that use the zoning api)